topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -170,140 +157,297 @@ To estimate sequence quality along all sequences, we now use [FastQC](https://ww
+> The following example is from a high quality DNA-Seq library. The biased Kmers near the start of the read likely are due to slight, sequence dependent efficiency of DNA shearing or a result of random priming:
 >
->     The presence of any overrepresented sequences in your library (such as adapter dimers) causes the Kmer plot to be dominated by the Kmers these sequences contain, and then it's not always easy to see if there are other biases present in which you might be interested.
+> ![Adapter Content](../../images/quality-control/kmer_content.png "Kmer content")
 >
->     The plot shows a cumulative percentage count of the proportion of your library which has seen each of the adapter sequences at each position. Once a sequence has been seen in a read it is counted as being present right through to the end of the read so the percentages you see will only increase as the read length goes on.
+> This module can be very difficult to interpret. RNA-seq libraries may have highly represented kmers that are derived from highly expressed sequences. To learn more about this plot, please check the [FastQC Kmer Content documentation](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/3%20Analysis%20Modules/11%20Kmer%20Content.html).
 >
 {: .details}
 
-
-> ### {% icon question %} Questions
+> ### {% icon details %} Specific problem for alternate library types
 >
-> Why is there a warning for the per-base sequence content and the per-sequence GC content graphs?
+> #### Small/micro RNA
+> 
+> In small RNA libraries, we typically have a relatively small set of unique, short sequences. Small RNA libraries are not randomly sheared before adding sequencing adapters to their ends: all the reads for specific classes of micro RNA will be identical. It will result in:
+> 
+> - Extremely biased per base sequence content
+> - Extremely narrow distribution of GC content
+> - Very high sequence duplication levels
+> - Abundance of overrepresented sequences
+> - Read through into adapters
 >
-> > ### {% icon solution %} Solution
-> > In the beginning of sequences, the sequence content per base is not really good and the percentages are not equal. For the GC content, the distribution is slightly shifted on the left, and too high.
-> {: .solution }
-{: .question}
-
+> #### Amplicon
+> 
+> Amplicon libraries are prepared by PCR amplification of a specific target. For example, the V4 hypervariable region of the bacterial 16S rRNA gene. All reads from this type of library are expected to be nearly identical. It will result in:
+>
+> - Extremely biased per base sequence content
+> - Extremely narrow distribution of GC content
+> - Very high sequence duplication levels
+> - Abundance of overrepresented sequences
```

Comment:
should have the same quality problems as the RADSeq, or are specific base caller parameters used?

Reply:
I never analyzed RADSeq data. Do you want to add the use case here?

Reply:
not necessarily